crates/types/src/protobuf.rs

```diff
@@ -111,16 +110,10 @@ pub mod node {
             in_response_to: Option<u64>,
         ) -> Self {
             let context = tracing::Span::current().context();
-            let span_context = if context.span().span_context().is_sampled() {
-                let mut span_context = SpanContext::default();
-                global::get_text_map_propagator(|propagator| {
-                    propagator.inject_context(&context, &mut span_context)
-                });
-
-                Some(span_context)
-            } else {
-                None
-            };
+            let mut span_context = SpanContext::default();
```

Comment:
`is_sampled` flag is already sent over via the span context to the receiving side so it's better to send the span_context always